driver-core/src/test/unit/com/mongodb/client/model/changestream/ChangeStreamDocumentSpecification.groovy

```diff
@@ -24,9 +24,34 @@ import spock.lang.Specification
 
 class ChangeStreamDocumentSpecification extends Specification {
 
+    def 'should initialize correctly'() {
```

Comment:
This test is basically the same as the one already here.  You can just append the when and then clauses to the bottom of that test, so that you don't have to copy the given clause

Reply:
Done